### PR TITLE
feat: create initial docs

### PR DESCRIPTION
**Pull request by kantorcodes**
Creates initial docs utilizing Docusaurus

### CODE REVIEW

File: docs/docs/demos.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Demos
+
+We've built two fully functional dApp and wallet examples to help you get started with Hedera Wallet Connect.
+
+- [React Demo](https://github.com/hgraph-io/hedera-wallet-connect/tree/main/demos/react-dapp)
+- [Typescript Demo](https://github.com/hgraph-io/hedera-wallet-connect/tree/main/demos/typescript)
```
Comment (tmctl) [inline docs/docs/demos.md:6]: might want to update to the hashgraph repo link

File: docs/docs/installation.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+### Using npm
+
+```bash
+npm install @hashgraph/hedera-wallet-connect @hashgraph/sdk @hashgraph/proto @hashgraph/hedera-wallet-connect @walletconnect/modal @walletconnect/qrcode-modal @walletconnect/utils @walletconnect/types @walletconnect/modal-core
+```
+
+### Using yarn
+
+```bash
+yarn add @hashgraph/hedera-wallet-connect @hashgraph/sdk @hashgraph/proto @hashgraph/hedera-wallet-connect @walletconnect/modal @walletconnect/qrcode-modal @walletconnect/utils @walletconnect/types @walletconnect/modal-core
```
Comment (tmctl) [inline docs/docs/installation.md:52]: do we need to list all of these?

**Review by tmctl — APPROVED** [review]
looking good!

File: docs/docs/installation.md (hunk 2 above)
Comment (kantorcodes) [reply docs/docs/installation.md:52]: > do we need to list all of these?

I believe so. When I developed `hashinals-wc` I did into issues without having a lot of these. I don't know 100% that we need all of them, but can w can likely omit `@walletconnect/types`.

**Pull request merged**